Sources/PrimerSDK/Classes/PCI/Tokenization View Models/FormsTokenizationViewModel/FormPaymentMethodTokenizationViewModel+FormViews.swift

```diff
@@ -433,7 +436,14 @@ extension FormPaymentMethodTokenizationViewModel {
             let pcfvc = PrimerInputViewController(navigationBarLogo: uiModule.navigationBarLogo,
                                                   formPaymentMethodTokenizationViewModel: self,
                                                   inputsDistribution: .horizontal)
-            inputs.append(contentsOf: makeInputViews())
+
+            let newInputs = makeInputViews()
+
+            // Append only inputs that are not already in the array
+            for newInput in newInputs where !self.inputs.contains(where: { $0 === newInput }) { // Use reference comparison
+                self.inputs.append(newInput)
+            }
+
```

Comment:
Can you expand on what this change is addressing?

Reply:
this relates to Scenario 2. in the Jira ticket @NQuinn27